main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -146,6 +146,8 @@ class Flix {
     "Core/Io/ZipInput.flix" -> LocalResource.get("/src/library/Core/Io/ZipInput.flix"),
     "Core/Io/ZipOutput.flix" -> LocalResource.get("/src/library/Core/Io/ZipOutput.flix"),
 
+    "Core/System/Environment.flix" -> LocalResource.get("/src/library/Core/System/Environment.flix"),
```

Comment:
In terms of namespace, we are currently just dumping everything in the root, and then we later plan to design what namespace should exist, so just move `Environment`  to the root (i.e. out of Core/System.